Demo/Cells/UserAdTableViewCell/UserAdCellDemoView.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  Copyright © 2020 FINN AS. All rights reserved.
+//
+
+import Foundation
+import FinniversKit
+
+class UserAdCellDemoView: UIView, Tweakable {
+
+    lazy var tweakingOptions: [TweakingOption] = [
+        TweakingOption(title: "Default") { self.style = .default },
+        TweakingOption(title: "Compact") { self.style = .compressed },
+    ]
+
+    private var style: UserAdTableViewCell.Style = .default {
+        didSet {
+            tableView.reloadData()
+        }
+    }
+
+    private let viewModels: [UserAdTableViewCellViewModel] = UserAdsFactory.createAds()
+
+    private lazy var tableView: UITableView = {
+        let tableView = UITableView(withAutoLayout: true)
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.register(UserAdTableViewCell.self)
+        tableView.separatorInset = .leadingInset(frame.width)
+        return tableView
+    }()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setup() {
+        addSubview(tableView)
+        tableView.fillInSuperview()
+    }
+
+}
+
+// MARK: - UITableViewDelegate
+
+extension UserAdCellDemoView: UITableViewDelegate {
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        tableView.deselectRow(at: indexPath, animated: true)
+    }
+}
+
+// MARK: - UITableViewDataSource
+
+extension UserAdCellDemoView: UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return viewModels.count
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeue(UserAdTableViewCell.self, for: indexPath)
+        cell.configure(with: style, model: viewModels[indexPath.row])
+        cell.remoteImageViewDataSource = self
+        cell.loadingColor = .toothPaste
+
+        return cell
+    }
+
+    func tableView(_ tableView: UITableView, willDisplay cell: UITableViewCell, forRowAt indexPath: IndexPath) {
+        if let cell = cell as? UserAdTableViewCell {
+            cell.loadImage()
+        }
+    }
+}
+
+extension UserAdCellDemoView: RemoteImageViewDataSource {
+    func remoteImageView(_ view: RemoteImageView, cachedImageWithPath imagePath: String, imageWidth: CGFloat) -> UIImage? {
+        return nil
+    }
+
+    func remoteImageView(_ view: RemoteImageView, loadImageWithPath imagePath: String, imageWidth: CGFloat, completion: @escaping ((UIImage?) -> Void)) {
+        guard let url = URL(string: imagePath) else {
+            completion(nil)
+            return
+        }
+
+        // Demo code only.
+        let task = URLSession.shared.dataTask(with: url) { data, _, _ in
```

Comment:
I've learned that these network calls should have some sleep included, so we don't mess with the snapshot tests.

From `FrontPageDemoView`:
```swift
// Demo code only.
let task = URLSession.shared.dataTask(with: url) { data, _, _ in
    usleep(50_000)
    DispatchQueue.main.async {
        if let data = data, let image = UIImage(data: data) {
            completion(image)
        } else {
            completion(nil)
        }
    }
}
```